src/components/AirbnbPreview.js

```diff
@@ -0,0 +1,36 @@
+import React from "react"
+import styled from 'styled-components';
+import scriptLoader from "react-async-script-loader";
+
+const PageWrapper = styled.div`
```

Comment:
if you want to define `PageWrapper` in a common components directory, you can do so and then include it in this component and the GoogleMaps component

Reply:
ok later